File: test_contest/P1165B.cpp
#include <iostream>
#include <bits/stdc++.h>
using namespace std;

int main()
{
  int n;
  cin >> n;
  int p[n];

  for (int i = 0; i < n; i++)
  {
    cin >> p[i];
  }

  sort(p, p + n);

  int res = 0;

  for (int i = 0; i < n; i++)
  {
    if (p[0] >= res + 1)
      res++;
  }

  cout << res;
  return 0;
}

File: test_contest/P1165B.py

import math
import sys
input = sys.stdin.readline


def inp():
    return int(input())


def invr():
    return list(map(int, input().split()))


n = inp()

p = invr()

p.sort()

res = 0

for i in range(n):
    if p[i] >= res + 1:
        res += 1

print(res)
